bears/vcs/git/GitCommitBear.py

```diff
@@ -61,25 +61,31 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
                                            allowed or not.
         """
         with change_directory(self.get_config_dir() or os.getcwd()):
-            stdout, stderr = run_shell_command('git log -1 --pretty=%B')
-
-        if stderr:
-            self.err('git:', repr(stderr))
+            stdout_head, stderr_head = run_shell_command(
+                "git log -1 --pretty=%B")
+            stdout_shortlogs, stderr_shortlogs = run_shell_command(
+                "git log --pretty=%s")
```

Comment:
Oh, I assumed that just splitting at new lines wouldn't be a problem even if it were say 10k ...

I'll try to come up with something better